lib/AST/Decl.cpp

```diff
@@ -410,6 +410,10 @@ template<typename Class>
 inline char checkSourceLocType(SourceLoc (Class::*)() const);
 inline TwoChars checkSourceLocType(SourceLoc (Decl::*)() const);
 
+template<typename Class>
+inline char checkSourceLocType(SourceLoc (Class::*)(bool) const);
+inline TwoChars checkSourceLocType(SourceLoc (Decl::*)(bool) const);
```

Comment:
It looks like you didn't end up using these? (They're the same as what's above.)

Reply:
They are difference for taking `bool` as function argument. We need these for `getLoc(bool)` and the above ones for `getLocFromSource()`.

Reply:
Oh, whoops. My mistake!